test/fft_utest.cpp

```diff
@@ -295,7 +295,7 @@ TYPED_TEST(FftTest, TestFftAdd) // NOLINT
 
         unsigned len = this->code_len;
         if (gf.card_minus_one() <= this->code_len) {
-            len = gf.weak_rand();
+            len = gf_n;
```

Comment:
Okay, thanks :)

Please fix also L68: https://github.com/scality/quadiron/pull/244/files#diff-594f3543c3c5e448ed6d59ccc64d2169R68
```
            const int t = 2 + gf.weak_rand() % (fft->get_n() - 1);
            const int n = t + gf.weak_rand() % (fft->get_n() - t + 1);
```

Reply:
This was to handle the case where `len = 2`, but if `len == 2` you'll fail while calling `taylor_expand` anyway, so this fix is not necessary, is it?

Reply:
It fixes another bug for `n` and `t`, as we want to have them such that `1 < t <= n`

Reply:
Done

Reply:
thanks